kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/MctsBeneficiaryUpdateServiceImpl.java

```diff
@@ -226,7 +189,7 @@ private MctsBeneficiary beneficiaryFromId(String mctsId, String stateMctsId) {
             beneficiary = mctsChildDataService.findByBeneficiaryId(mctsId);
         }
 
-        // then try the State ID
+        // then try the State ID // What on earth is a stateMctsId??
```

Comment:
the dinosaur is already dead... and besides, state mcts id is not a real thing. the earth is only 6000yrs old